### PR TITLE
txdag transportion

### PR DESCRIPTION
**Pull request by andyzhang2023**

### CODE REVIEW

File: miner/worker.go [hunk 1]
```diff
@@ -1459,3 +1482,52 @@ func signalToErr(signal int32) error {
 		panic(fmt.Errorf("undefined signal %d", signal))
 	}
 }
+
+// generateDAGTx generates a DAG transaction for the block
+func (w *worker) generateDAGTx(env *environment) (*types.Transaction, error) {
+	// @TODO this is a placeholder for the txDAG transaction sender
+	privateKeyHex := "05305d9000ae59abb613799237e87885dd122bafd71beffb60cc15ebf95d1d1f" //0x0fC7B89DdE80018CaF9eEd78757f996d9d3a25Aa
```
Comment (galaio) [inline miner/worker.go:1489]: Do from and to have any special meaning?

File: miner/worker.go [hunk 2]
```diff
@@ -1459,3 +1482,52 @@ func signalToErr(signal int32) error {
...
+	toAddress := "0x559fb1e8707DA1F6De2A3eA5C05db84eDb13232d"
+
+	statedb, err := w.chain.State()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get state db, err: %v", err)
+	}
+
+	// get txDAG data from the stateDB
+	txDAG, _ := statedb.MVStates2TxDAG()
```
Comment (galaio) [inline miner/worker.go:1498]: Does there need to add self-dependency?

File: miner/worker.go (hunk 1 above)
Comment (andyzhang2023) [reply miner/worker.go:1489]: No，just two normal addresses.

File: miner/worker.go (hunk 2 above)
Comment (andyzhang2023) [reply miner/worker.go:1498]: Yes, it should be better to have one.

**Pull request closed without merging**